runtime/doc/options.txt

```diff
@@ -5144,6 +5144,12 @@ A jump table for the options with a short description can be found at |Q_op|.
 
 	This option cannot be set from a |modeline| or in the |sandbox|, for
 	security reasons.
+							*shell-powershell*
+	To use powershell (on Windows): >
+		set shell=powershell shellquote=\" shellpipe=\| shellredir=>
+		set shellcmdflag=\ -NoLogo\ -NoProfile\ -ExecutionPolicy\ RemoteSigned\ -Command
+		let &shellxquote=' '
```

Comment:
I use powershell with `set shellxquote=` with no issues. Is there an escape or concat issue if this is unset?

Reply:
I think you mentioned this already, but we haven't updated helpers.lua yet so I again copy-pasted it in error.